testdata/p4_16_samples/issue4656_const_fold_generic_switch_label_expr.p4

```diff
@@ -0,0 +1,24 @@
+#include <core.p4>
+
+extern void foo();
+extern void bar();
+extern bit<8> baz();
+action a(){}
+action b(){}
+control c() {
+    table t  {
+        actions = { a ; b;  }
+    }
+    apply {
+        switch(baz()) {
+            1 + 2 == 3 ? 1 : 2 : { foo(); }
```

Comment:
I am just surprised this is not syntactically ambiguous, the 2 `:` look quite weird :-). (This is not an issue with the PR/test, I'm just surprised the grammar allows it).

Reply:
A single token of lookahead in Bison is apparently enough for it to figure it out, especially from the context that this is inside of a `switch` statement.